doc/short_vignettes/selective_sweep.md

```diff
@@ -0,0 +1,152 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(selective_sweeps)=
+
+# Selective sweeps
+
+```{code-cell} python
+---
+tags: ['hide-input']
+---
+
+import fwdpy11
+import numpy as np
+import msprime
+```
+
+```{code-cell} python
+import fwdpy11.conditional_models
+import fwdpy11.tskit_tools
+```
+
+
+```{code-cell} python
+---
+tags: ['hide-input']
+---
+
+def setup(prune_selected=False):
+    # Dropping mutations requires existing
+    # ancestry, which we can get either
+    # from a burn-in or from msprime.
+    initial_ts = msprime.sim_ancestry(
+        samples=500,
+        population_size=1000,
+        recombination_rate=1e-1,
+        random_seed=43215,
+        sequence_length=1.0,
+    )
+
+    # Build the pop from msprime output
+    pop = fwdpy11.DiploidPopulation.create_from_tskit(initial_ts)
+
+    # Set up basic model parameters
+    pdict = {
+        "recregions": [fwdpy11.PoissonInterval(0, 1, 1e-1)],
+        # Here, 2 means that fitness is multiplicative
+        # over 1, 1+hs, 1+2s.
```

Comment:
Should this be `1, 1+hs, 1+s` or `1, 1+2hs, 1+2s`, with `h=0.5` meaning additive selection? (maybe I'm getting confused with how it interacts with `Multiplicative(2.0)`...)

Reply:
It is always 1+hs, so h changes range as scaling changes. yes, that is likely a sticking point...